src/collision/collision.c

```diff
@@ -0,0 +1,48 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   collision.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: ycarro <ycarro@student.42.fr>              +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/02/23 14:59:18 by ycarro            #+#    #+#             */
+/*   Updated: 2023/02/23 17:12:59 by ycarro           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <collision.h>
```

Comment:
does this work? shouldn't we use :
`#include <collision.h> -> #include "collision.h"`

Reply:
True